src/parser/postgresparser.cpp

```diff
@@ -556,42 +555,42 @@ std::unique_ptr<AbstractExpression> PostgresParser::ColumnRefTransform(ColumnRef
 }
 
 // Postgres.A_Const -> terrier.ConstantValueExpression
-std::unique_ptr<AbstractExpression> PostgresParser::ConstTransform(A_Const *root) {
+std::unique_ptr<AbstractExpression> PostgresParser::ConstTransform(ParseResult *parse_result, A_Const *root) {
   if (root == nullptr) {
     return nullptr;
   }
-  return ValueTransform(root->val_);
+  return ValueTransform(parse_result, root->val_);
 }
 
 // Postgres.FuncCall -> terrier.AbstractExpression
-std::unique_ptr<AbstractExpression> PostgresParser::FuncCallTransform(FuncCall *root) {
-  // TODO(WAN): change case?
+std::unique_ptr<AbstractExpression> PostgresParser::FuncCallTransform(ParseResult *parse_result, FuncCall *root) {
+  // TODO(WAN): Check if we need to change the case of this.
   std::string func_name = reinterpret_cast<value *>(root->funcname_->head->data.ptr_value)->val_.str_;
 
   std::unique_ptr<AbstractExpression> result;
   if (!IsAggregateFunction(func_name)) {
     // normal functions (built-in functions or UDFs)
     func_name = (reinterpret_cast<value *>(root->funcname_->tail->data.ptr_value))->val_.str_;
-    std::vector<std::shared_ptr<AbstractExpression>> children;
+    std::vector<std::unique_ptr<AbstractExpression>> children;
 
     if (root->args_ != nullptr) {
       for (auto cell = root->args_->head; cell != nullptr; cell = cell->next) {
         auto expr_node = reinterpret_cast<Node *>(cell->data.ptr_value);
-        children.emplace_back(ExprTransform(expr_node));
+        children.emplace_back(ExprTransform(parse_result, expr_node, nullptr));
```

Comment:
Same as above, does it need an explicit move call?

Reply:
See above.